### ISSUE TITLE
[bug]: Estimate points don't return by API

### ISSUE THREAD

**Issue by mmospanenko** [makeplane/plane#5213]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

https://my.app/api/v1/workspaces/proj_slug/projects/proj_id/issues/issue_id/ returns wrong Estimate Ponit data.

'point' is null, but value is set. 

Also there is not 'links' value in the response.

I use token for auth

### Steps to reproduce

1. create issue with Point
2. get this issue via api

### Environment

Production

### Browser

Google Chrome

### Variant

Self-hosted

### Version

0.22

**Comment by NarayanBavisetti** [comment]
Hey @mmospanenko , we recently updated our estimation system and now use a field called `estimate_point` to store the ID of the estimate point. Currently, we don't have an external API to list the estimates, but we plan to add one in the future.
would love to know your use case for how you use estimates

**Comment by mmospanenko** [comment]
yes, this is about new version. There is a 'point' property, it should ve value (number). I use it for syncing PM data with BI systems for analytics, so I need to get ETA in hours in any way. Yes, there is a `estimate_point` (FK on estimate object) and also I see `point`, that contains its value number (we use it as hours). And the issue is that this `point` is null sometimes when we have selected Estimate.

Will be great to have more usable ETA system (Expected, Estimated, Spent, Extra hours), but now its ok, just fix fill be great, I know you're working on refactoring this.

Thanks)

**Comment by NarayanBavisetti** [comment]
![Image](https://github.com/user-attachments/assets/27deb2ca-cd95-4678-849e-da7e90c3af89)

@mmospanenko  you can use `expand=estimate_point` in the query to get the value of the estimate point.

**Comment by NarayanBavisetti** [comment]
This issue has been closed as it has been resolved. If you still face any issue, you can reopen it.

**Comment by webbugt** [comment]
This should be documented in the API docs :]
